### PR TITLE
Updates about copy

### PR DESCRIPTION
**Pull request by jorgelainfiesta**
## Motivation

Charles distilled the project's values and objectives, and now we're taking care of structuring the copy in a more snappy way.

## Approach 

I edited the about page with Olson as customary for the Frontside blog. I think he did a great job! 

[link to page preview](https://deploy-preview-734--bigtest.netlify.app/bigtest/about)

### CODE REVIEW

File: website/src/pages/about.mdx [hunk 1]
```diff
@@ -1,124 +1,40 @@
...
-wait times to observable states, interactors naturally use the minimum
-amount of synchronization time while performing actions and
-assertions. Finally, because interactors are strongly associated with
-both the type and properties of the UI elements they actuate
-(e.g. Button, Checkbox, Input, ...), they can provide the next level
-in error messaging when something goes wrong, or an assertion fails.
+The length of the testing feedback loop is not determined by just how fast the test can report it succeeds or fails. Of course a test suite that can validate 800 assertions per minute has twice the throughput as a project that can only validate 400. But the critically important aspect of the speed of the testing feedback loop is how quickly and accurately you are able to understand why a test failed. A test that takes only 5 milliseconds to run but fails with an error message so inscrutable that it takes a further 90 minutes to figure out why is in fact excruciatingly slow. In order to make testing feedback loops as short as possible, BigTest components run quickly and provide failures that are precise and helpful.
```
Comment (cowboyd) [inline website/src/pages/about.mdx]:
```suggestion
The length of the testing feedback loop is not determined by just how fast the test can report success or failure. Of course a test suite that can validate 800 assertions per minute has twice the throughput as a project that can only validate 400. But the critically important aspect of the speed of the testing feedback loop is how quickly and accurately you are able to understand why a test failed. A test that takes only 5 milliseconds to run but fails with an error message so inscrutable that it takes a further 90 minutes to figure out why is in fact excruciatingly slow. In order to make testing feedback loops as short as possible, BigTest components run quickly and provide failures that are precise and helpful.
```

**Review by cowboyd — COMMENTED** [review]
These are really awesome!

The only change that didn't quite enhance the vibe we're going for was this statement:

> A test that is both fast and precise is not enough; it has to measure something of value as well. That’s not to say a React component rendering its props correctly isn’t critical; it’s just that a truly useful test would also test the component used in a workflow that correctly transfers money from one user account to another. 

I just don't think it really captures the juxtaposition of a test that covers very little value vs one that does. Maybe useful is not the right word?

**Comment by jorgelainfiesta** [comment]
@cowboyd  how about something like this?

> That's not to say a React component rendering its props correctly isn't useful; it's just that a truly valuable test would act as insurance for your business, for instance, checking the workflow of correctly transferring money from one user account to another.

**Comment by cowboyd** [comment]
> @cowboyd how about something like this?
> 
> > That's not to say a React component rendering its props correctly isn't useful; it's just that a truly valuable test would act as insurance for your business, for instance, checking the workflow of correctly transferring money from one user account to another.

I like it, although is that two uses of "truly valuable?"

**Comment by jorgelainfiesta** [comment]
True! I updated the PR with this section text

> A test that is both fast and precise is not enough; it has to measure something of value as well. That's not to say a React component rendering its props correctly isn't useful; it's just that great tests would act as insurance for your business, for instance, checking the workflow of correctly transferring money from one user account to another. At Google they’ve [come to the realization][5] that "unit test," "functional test" and "end-to-end test" are arbitrary designations and what really matters is the relative size of a test. The larger a test, the more systems it integrates and thus the more business value it protects. The only hitch of course is that test size is in direct conflict with feedback loop speed.

> The BigTest project approaches every piece of its platform from the perspective of empowering development teams to master this tension and optimize testing feedback loops within the context of truly valuable tests.

**Review by cowboyd — APPROVED** [review]
Chef's kiss 👨🏻‍🍳 👌🏻 💋

**Pull request merged**